packages/widget-scholia/src/Scholion.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="line">
+    <span
+      class="lemma"
+      @click="onScholionClick(line)"
+      :class="{ highlight: isScholionHiglighted }"
+    >{{ line.lemma }}</span>
+    <span class="comment"> {{ line.comment }}</span>
+  </div>
+</template>
+
+
+<script>
+import { MODULE_NS, SELECT_SCHOLION, SELECT_LINE } from '@scaife-viewer/store';
+
+export default {
+  name: 'Scholion',
+  props: {
+    line: Object
+  },
+  computed: {
+    isScholionHiglighted() {
+      const highlightedScholion = this.$store.getters[`${MODULE_NS}/selectedScholion`];
+
+      // NB: After the highlightedScholion is set for the first time,
+      // it becomes an Observer --- that's why we check for null/undefined
+      // of highlightedScholion.idx as well.
+      if (highlightedScholion !== null && highlightedScholion.idx != null) {
```

Comment:
I'm not sure about checks like this one -- I'm not familiar with the idioms for dealing with Vuex's Observer objects, but this feels a little hacky to me.